app/grants/templates/grants/cart/eth.html

```diff
@@ -3,408 +3,260 @@
-                <div class="btn btn-sm btn-gc-blue font-title font-weight-normal ml-xl-2 px-4"
-                  :class="{inactive: ![0,5,10,15].includes(gitcoinFactorRaw), 'badge-inactive': gitcoinFactorRaw != 15}"
-                  @click="gitcoinFactorRaw=15">
-                  15%</div>
-                <div class="btn btn-sm mx-3 btn-gc-blue font-title font-weight-normal ml-xl-2 px-4"
-                  :class="{inactive: ![0,5,10,15].includes(gitcoinFactorRaw), 'badge-inactive': gitcoinFactorRaw != 0}"
-                  @click="gitcoinFactorRaw=0">
-                  0%</div>
-                <div class="d-inline-block">
-                  <span class="font-caption md:pl-3">Custom %</span>
-                  <input style="width: 75px;" class="border btn font-title font-weight-normal ml-xl-2 pl-0 text-center mr-2"
-                    type="number" min="0" max="99" step="any" id="gitcoin-grant-input-amount"
-                    name="gitcoin-grant-input-amount" class="form__input text-center"
-                    :class="{inactive: [0,5,10,15].includes(gitcoinFactorRaw)}" value="5"
-                    v-model="gitcoinFactorRaw"
-                  >
-                  {% include '../heart.html' %}
-                </div>
-              </div>
-              <br/>
+  <!-- Summary -->
+  <div>
+    <h4 class="col text-dark font-weight-bold p-0">Summary</h4>
 
-              <div class="flex-container text-left grant-header-row black m-1 p-1">
-                <div class="w-100">
-                  <div class="flex-container" style="justify-content: start">
-                    <input type="checkbox" id="include_for_clr" name="include_for_clr" v-model="include_for_clr" style="margin-right:1rem">
-                    <label class="font-smaller-1 darker-gray mb-0 pb-0" for="include_for_clr">
-                      Include this contribution to CLR matching
-                    </label>
-                  </div>
-                </div>
-              </div>
-            </div>
-            <!-- changed here -->
-          </div>
-        </div>
-      </div>
-    </div>
+    <ul>
+      <li>You are contributing [[donationsToGrantsString]]</li>
+      <li>You are additionally contributing [[donationsToGitcoinString]] to the Gitcoin Maintainer Grant</li>
+      <li>
+        Note: The exact checkout flow will depend on whether you use standard checkout or zkSync.
+        <a href="https://github.com/gitcoinco/web/blob/master/docs/GRANTS.md" target="_blank">Read how this works</a>.
+      </li>
+    </ul>
   </div>
 
-  {% comment %} Summary {% endcomment %}
-  <div class="flex-container font-bigger-1 text-left grant-header-row black"
-    style="font-weight: bold; margin-top: 1rem;">
-    <div style="width: 100%">
-      <div class="flex-container" style="justify-content: start">
-        SUMMARY
+  <!-- Checkout recommendation -->
+  <div class="row justify-content-end mb-3">
+    <small class="col-auto bg-light p-2">
```

Comment:
Thank you for checking in on this @mds1: `.bg-light` is used throughout the codebase, and I'm not sure what colour we will need to settle on for the rest of the design to make sense with minimal changes, I think the safest option for now would be `.bg-lightpurple`.

Reply:
Thanks, updated to use `.bg-lightpurple` in 146d26a25, and will use that going forward